src/appengine/handlers/cron/grouper.py

```diff
@@ -31,6 +31,7 @@
 GROUP_MAX_TESTCASE_LIMIT = 25
 
 VARIANT_CRASHES_IGNORE = ['Out-of-memory', 'Timeout']
+VARIANT_THRESHOLD_PERCENTAGE = 0.05
```

Comment:
Let's make this higher. 5% feels a bit low. Maybe 20%?  

The case we saw was of a startup crash in a bad job type, where in theory **every** single testcase will result in the same result in that bad job type. 20% is still fairly conservative here.